### DISCUSSION TITLE
Backend not load

### DISCUSSION THREAD

**Discussion by f0r7y** [netbox-community/netbox#13763] [Help Wanted!]
Hi
after upgrading to version 3.6.1, the admin section looks like this
![Screenshot 2023-09-14 alle 15 14 46](https://github.com/netbox-community/netbox/assets/110922392/f4b123f0-a25e-4279-a587-aafa98ed2262)
I'am unable to manage anythings

**Comment by candlerb** [comment]
That looks roughly correct. Most of the admin functions have moved to the main Netbox UI in Netbox 3.6, see the "Admin" section in the left-hand main navigation bar where you now manage users, API tokens, config revisions etc. This is clearly called out in the [v3.6.0 release notes](https://github.com/netbox-community/netbox/releases/v3.6.0).

However, the old admin page does still exist and has a few remaining functions. Here's what I see:
![image](https://github.com/netbox-community/netbox/assets/44789/67b66fea-334f-42b5-a1cc-e94265eac4b4)

In your case, you seem to have the same as me except for that big grey area which says "Toggle theme"; I don't know where that comes from.

Try a shift-reload in your browser.

Otherwise, maybe something went wrong during your upgrade? Did you use a git pull or a zip/tar upgrade? If the latter, did you unpack a fresh tarball into an empty directory then change the symlink, as the instructions say? (Overwriting an existing install will not work).  Did you run `./upgrade.sh` and were there any errors reported?
